src/modules/user/domain/events/events-success-domain/user-archived.event.ts

```diff
@@ -0,0 +1,6 @@
+export class UserArchivedEvent {
+  constructor(
+    public readonly uuid: string,
+    public readonly deletedAt: Date,
```

Comment:
No manejes deletedAt maneja archivedAt
```suggestion
    public readonly archivedAt: Date,
```